config/wpcalypso.json

```diff
@@ -131,6 +131,7 @@
 		"signup/stepper-flow": true,
 		"signup/woo-verify-email": false,
 		"signup/theme-preview-screen": true,
+		"signup/standard-theme-v13n": true,
```

Comment:
Is there any reason we want to enable it in `wpcalypso`? 🤔

Reply:
I forgot the reason but I think we don't need to enable it now